src/drivers/Cst816s.cpp

```diff
@@ -56,28 +57,33 @@ bool Cst816S::Init() {
   static constexpr uint8_t irqCtl = 0b01110000;
   twiMaster.Write(twiAddress, 0xFA, &irqCtl, 1);
 
+  // Disable auto-reset after 5s of no gesture
+  // The reset kills the current touch point, so paint/pong etc breaks
+  twiMaster.Write(twiAddress, 0xFB, 0, 1);
+
   return true;
 }
 
 Cst816S::TouchInfos Cst816S::GetTouchInfo() {
   Cst816S::TouchInfos info;
-  uint8_t touchData[7];
+  std::array<uint8_t, 6> touchData {};
 
-  auto ret = twiMaster.Read(twiAddress, 0, touchData, sizeof(touchData));
+  constexpr uint8_t addressOffset = 1;
+  auto ret = twiMaster.Read(twiAddress, addressOffset, touchData.data(), sizeof(touchData));
```

Comment:
now we can use .size()

```suggestion
  auto ret = twiMaster.Read(twiAddress, addressOffset, touchData.data(), touchData.size());
```